src/java.base/share/classes/java/lang/System.java

```diff
@@ -324,6 +327,22 @@ private static void checkIO() {
     private static native void setOut0(PrintStream out);
     private static native void setErr0(PrintStream err);
 
+    // Remember original System.err. setSecurityManager() warning goes here
+    private static PrintStream oldErrStream = null;
```

Comment:
I assume this should needs to be volatile and @Stable. I think we need a better name for it too.

Reply:
Will add the modifiers. How about "originalErr"?